integration-test/common/src/test/java/com/tencent/rss/test/HealthCheckCoordinatorGrpcTest.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Tencent is pleased to support the open source community by making
+ * Firestorm-Spark remote shuffle server available.
+ * Copyright (C) 2021 THL A29 Limited, a Tencent company.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * https://opensource.org/licenses/Apache-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.tencent.rss.test;
+
+import com.google.common.collect.Sets;
+import com.google.common.util.concurrent.Uninterruptibles;
+import com.tencent.rss.client.request.RssGetShuffleAssignmentsRequest;
+import com.tencent.rss.client.response.ResponseStatusCode;
+import com.tencent.rss.client.response.RssGetShuffleAssignmentsResponse;
+import com.tencent.rss.common.util.Constants;
+import com.tencent.rss.coordinator.CoordinatorConf;
+import com.tencent.rss.coordinator.ServerNode;
+import com.tencent.rss.server.ShuffleServerConf;
+import com.tencent.rss.storage.util.StorageType;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
```

Comment:
Shoudle the empty line be between line 29 and 30?

Reply:
You can increase a rule in checkstyle.xml. Currently, we don't have clear rule.

Reply:
So we could remove the extra empty line if no clear imports grouping rule or purpose here.